mindmeld/components/dialogue.py

```diff
@@ -129,7 +129,9 @@ def __init__(self, dialogue_state, **kwargs):
                     raise ValueError(msg.format(single, plural))
                 elif single in kwargs and isinstance(kwargs[single], str):
                     resolved[plural] = {kwargs[single]}
-                elif plural in kwargs and isinstance(kwargs[plural], (list, set, tuple)):
+                elif plural in kwargs and isinstance(
```

Comment:
I think this file is just applying black right?

Reply:
That's right, fixing lint errors that weren't caught before